stamina-core/src/main/scala/stamina/migrations.scala

```diff
@@ -55,20 +45,36 @@ package migrations {
    * }}}
    *
    *  @tparam R The type of raw data being migrated. In the JSON implementation this would be `JsValue`.
-   *  @tparam V The "current" version of this Migrator, i.e. it can migrate values from V1 to this version or any version in between.
+   *  @tparam V The "current" version of this Migrator.
    */
-  class Migrator[R, V <: Version: VersionInfo] private[stamina] (migrations: Map[Int, Migration[R]] = Map.empty) {
-    def canMigrate(fromVersion: Int): Boolean = migrations.contains(fromVersion)
+  class Migrator[R, V <: Version: VersionInfo] private[stamina] (forwardMigrations: Map[Int, Migration[R]] = Map.empty, backwardMigrations: Map[Int, Migration[R]] = Map.empty) {
+    def canMigrate(fromVersion: Int, toVersion: Int): Boolean = forwardMigrations.contains(fromVersion) && forwardMigrations.contains(toVersion)
```

Comment:
Is this correct? It's not obvious to me that we now also need `forwardMigrations.contains(toVersion)`, and shouldn't this also take into account that we can support backwards migrations?